### ISSUE TITLE
Add support for Ripple panic button

### ISSUE THREAD

**Issue by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [ccrama/Slide#3127]
Currently Aegis is not compatible with Ripple, it would be really nice and helpful for both projects if support would be added. Aegis would have an extra layer of security and Ripple would have one more app that has support, and this would incentivize Ripple's devs to continue with their work.

It would be even greater it you could add a setting to delete the entries, not just to hide the app when triggered. A lot of apps have support for it but only enable this option, which is kind of lame since it can have a bigger potential.

Thanks!

**Comment by Tensounder54** [comment]
Yeah, this is something that we can look into. Is there an API for ripple or something?

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
> Yeah, this is something that we can look into. Is there an API for ripple or something?

I'm not a developer and I don't really know how to search around the GH repositories, but here's the source, so I'm sure there has to be something in it

https://github.com/guardianproject/ripple

----

By the way, if you look at my edit you'll see that I have changed Aegis name for your app's name and some other stuff, that's because I'm trying to convince as much developers as I can to implement it.
Also, just in case you think the project is dead, it isn't. They have reached stable last year and since they haven't found bugs or new features it hasn't been updated, you can read more about this here: https://github.com/guardianproject/ripple/issues/13

**Comment by eighthave** [comment]
this is the relevant info for an app like this: https://guardianproject.info/code/panickit, Ripple is just one app in the PanicKit ecosystem.
